docs/UserGuide.md

```diff
@@ -218,37 +224,35 @@ Scroll through your history of commands using the `↑` or `↓` arrow keys.
 
 <br>
 
-### Edit a person : `edit INDEX ...` <a name="edit"></a>
+### Edit a contact : `edit INDEX ...` <a name="edit"></a>
```

Comment:
Thanks for your efforts in modifying this section, but I don't think you had to modify anything related to `edit` as I was assigned to reorganise any `edit`-related fields under my PR #204. Any changes made here will be superseded by #204. Affected lines are (wrt original version): 221 to 368. @david-eom maybe you can weigh in?

Reply:
Yep I agree w this! If this part conflicts with the UG-final pls keep the UG-final instead, I think Stanley has alr fixed all edit-related sections.